packages/widgets/src/lib/components/InferenceWidget/InferenceWidget.svelte

```diff
@@ -79,9 +79,11 @@
 	};
 
 	$: widgetComponent =
-		model.pipeline_tag && model.pipeline_tag in WIDGET_COMPONENTS
-			? WIDGET_COMPONENTS[model.pipeline_tag as keyof typeof WIDGET_COMPONENTS]
-			: undefined;
+		model.tags && "conversational" in model.tags
```

Comment:
```suggestion
		model.tags?.includes("conversational")
```

this is not Python:)